sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -164,6 +180,63 @@ trait GpuGenerator extends GpuUnevaluable {
   }
 }
 
+case class GpuReplicateRows(children: Seq[Expression]) extends GpuGenerator {
```

Comment:
You are getting a compile error in some cases.

```
Error: :29:25.509Z] [ERROR] [Error] /home/jenkins/agent/workspace/jenkins-rapids_premerge-github-3584/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala:183: class GpuReplicateRows needs to be abstract, since method withNewChildrenInternal in class TreeNode of type (newChildren: IndexedSeq[org.apache.spark.sql.catalyst.expressions.Expression])org.apache.spark.sql.catalyst.expressions.Expression is not defined
```

I think because of the shim layer you need to have this 

```
extends GpuUnevaluableUnaryExpression with GpuGenerator {
```

Reply:
fixed this compile error.